var/spack/repos/builtin/packages/feq-parse/package.py

```diff
@@ -12,10 +12,14 @@ class FeqParse(CMakePackage):
        provided as strings."""
 
     homepage = "https://github.com/FluidNumerics/feq-parse"
-    url      = "https://github.com/FluidNumerics/feq-parse/archive/v1.0.0.tar.gz"
+    url      = "https://github.com/FluidNumerics/feq-parse"
+    baseurl  = "https://github.com/FluidNumerics/feq-parse"
+    git      = "https://github.com/FluidNumerics/feq-parse.git"
 
-    maintainers = ['schoonovernumerics']
+    maintainers = ['fluidnumerics-joe']
 
-    version('1.0.2', sha256='1cd1db7562908ea16fc65dc5268b654405d0b3d9dcfe11f409949c431b48a3e8')
+    version('v1.1.0', tag='v1.1.0')
+    version('v1.0.2', tag='v1.0.2')
```

Comment:
you don't have tarball releases anymore?

Reply:
> you don't have tarball releases anymore?

@fluidnumerics-joe ?

Reply:
I still have tar-ball releases. Are the tar's preferred for some reason ?

Reply:
Actually, I don't know, but tarballs is the established source format in almost all spack packages.

Since it's a cmake package, there is no autoreconf to run on the source before build, so in this case there is no drawback from that.

OTOH, a git download means a git clone, with a .git directory somewhere instead of a tarball as source.
To archive that .git directory for a source mirror, it would have to be archived into a tarball currently if that's supported by spack.

So I guess, tarballs are just more source-mirror-friendly, as the source mirror of a tarball just indexes them by the sha256, even locally, while I don't know how spack handles caching and mirroring git clones. So I'd also be inclined towards tarballs if they are available.

Reply:
They're preferred because tags are not fixed to a commit and there is no SHA-sum to verify the integrity of the download. I don't think we mirror sources of a git tag indeed.